components/fieldgroups/experience-event/experienceevent-inappmessage-tracking.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/experienceevent-inappmessage-tracking",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Inapp message tracking",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Inapp message interaction tracking details",
+  "definitions": {
+    "experienceevent-inappmessage-tracking": {
+      "properties": {
+        "xdm:inappMessageTracking": {
```

Comment:
@iewilliam , you are adding a root level property "inappMessageTracking" in the experience-event schema. If there could be more ways in future to track a message or message response, we will have to add another root level property with a new prefix for message tracking. Can we just name it as "messageTracking" ?

Reply:
This already happened unfortunately, the 2 relevant type of mobile messaging we are tracking is push notification and inapp message. Push notification happened first and it already has its own top level field. Since we can't rename mixins, we can't simply reuse that mixin.